From b2b3fa0fdcb20ac47764ff5c43e6db016935f0dc Mon Sep 17 00:00:00 2001
From: Miouyouyou Myy <myy@miouyouyou.fr>
Date: Fri, 21 Jul 2017 00:00:59 +0000
Subject: [PATCH 1/4] dw_mmc-rockchip: Fixed typos which led to more typos

The dw_mmc-rockchip driver need 2 clocks named :
- "ciu-drive" and "ciu-sample"
However, the error messages spelt these clocks :
- "ciu_drv" and "ciu_sample"
Which led to some typos in various Rockchip DTSI files.

Signed-off-by: Miouyouyou Myy <myy@miouyouyou.fr>
---
 drivers/mmc/host/dw_mmc-rockchip.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/mmc/host/dw_mmc-rockchip.c b/drivers/mmc/host/dw_mmc-rockchip.c
index a3f1c2b..aaed6f9 100644
--- a/drivers/mmc/host/dw_mmc-rockchip.c
+++ b/drivers/mmc/host/dw_mmc-rockchip.c
@@ -282,11 +282,11 @@ static int dw_mci_rk3288_parse_dt(struct dw_mci *host)
 
 	priv->drv_clk = devm_clk_get(host->dev, "ciu-drive");
 	if (IS_ERR(priv->drv_clk))
-		dev_dbg(host->dev, "ciu_drv not available\n");
+		dev_dbg(host->dev, "ciu-drive not available\n");
 
 	priv->sample_clk = devm_clk_get(host->dev, "ciu-sample");
 	if (IS_ERR(priv->sample_clk))
-		dev_dbg(host->dev, "ciu_sample not available\n");
+		dev_dbg(host->dev, "ciu-sample not available\n");
 
 	host->priv = priv;
 
-- 
2.10.2

